server/README.md

```diff
@@ -1,8 +1,32 @@
-## How to see our database?
+## How run the backend?
+
+Go into our server folder.
+`cd server`  
+Install dependencies
+`yarn install`
+
+Configure `.env` file.
+
+1. Create a new file `.env` in the `server` folder at the root level.
+2. Go to discord's secret channel. Paste the contents inside the `.env`
+
+Actually run the backend  
+`nodemon` (if you are developing)  
+`yarn start` (if we are deploying I guess?)  
+Now the following link : http://localhost:5001/api/user
+
+## How to see our databases and how to use your own database?
+
+To see some particular database, whether it is the shared one, of your own, follow the steps below.
 
 1. Log in to Mongo Atlas
```

Comment:
We can add the link for easy access and mention where to get login details: 

1. Log in to [Mongo Atlas](https://account.mongodb.com/account/login?signedOut=true) (refer to discord's [secret](https://discord.com/channels/1106050152587874364/1110797187463512116) channel for user details)